pkg/backend/filestate/backend.go

```diff
@@ -827,27 +870,83 @@ func (b *localBackend) getLocalStacks() ([]tokens.Name, error) {
 	}
 
 	for _, file := range files {
-		// Ignore directories.
+
 		if file.IsDir {
-			continue
+			projName := objectName(file)
+			// If this isn't a valid Name it won't be a project directory, so skip it
+			if !tokens.IsName(projName) {
+				continue
+			}
+
+			projectFiles, err := listBucket(b.bucket, filepath.Join(path, projName))
```

Comment:
If there was an API to listBucket selectively with a wildcard would that be more appropriate? Thinking of buckets with many objects. Perhaps we never have those in the backend.

Reply:
It looks like you can ask for everything underneath (just don't pass a delimiter to the query, only a prefix), then we could do all the filtering and splitting client side. I think I'd rather keep this simple version for now but I'll add a TODO to revisit this to see if we can do less bucket API calls.